config.yaml

```diff
@@ -1,5 +1,7 @@
 extensions:
   health_check:
+    endpoint: "localhost:13133"
+    path: "/health/status"
```

Comment:
This is making explicit the default values, right?  Do we need to include them?

Reply:
Thanks. Updated.